### PR TITLE
[RTM] add error for model path type

### PR DESCRIPTION
**Pull request by AntoineLeroy**
add an error if the type of the model path is not correct + new example to test model path type

- - -
This change is [Reviewable](https://reviewable.io/reviews/pyomeca/bioptim/605)

### CODE REVIEW

File: bioptim/interfaces/biorbd_model.py [hunk 1]
```diff
@@ -9,8 +9,10 @@ class BiorbdModel:
     def __init__(self, bio_model: str | biorbd.Model):
         if isinstance(bio_model, str):
             self.model = biorbd.Model(bio_model)
-        else:
+        elif isinstance(bio_model, biorbd.Model):
             self.model = bio_model
+        else:
+            raise RuntimeError("Type must be a string or a biorbdmodel")
```
Comment (pariterre) [inline bioptim/interfaces/biorbd_model.py:15]: biorbd.Model

Comment (pariterre) [reply bioptim/interfaces/biorbd_model.py:15]: str

Comment (pariterre) [reply bioptim/interfaces/biorbd_model.py:15]: 'str' 'biorbd.Model'

File: tests/test_biorbd_model.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    from bioptim.examples.getting_started import pendulum as ocp_module
+
+    bioptim_folder = os.path.dirname(ocp_module.__file__)
+    model_path = "/models/pendulum.bioMod"
+    BiorbdModel(bioptim_folder + model_path)
+
+    BiorbdModel(biorbd.Model(bioptim_folder + model_path))
+
+    with pytest.raises(RuntimeError, match="Type must be a string or a biorbdmodel"):
+        BiorbdModel(1)
```
Comment (pariterre) [inline tests/test_biorbd_model.py:25]: `BiorbdModel([])` ?

**Review by pariterre — CHANGES_REQUESTED** [review]

**Review by AntoineLeroy — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pyomeca/bioptim/605)* status: 0 of 2 files reviewed, 2 unresolved discussions (waiting on @pariterre)

___
*[`bioptim/interfaces/biorbd_model.py` line 15 at r1](https://reviewable.io/reviews/pyomeca/bioptim/605#-NMigazvlHFKgkTAW1Ys-r1-15:-NMjQo0v5hIdSYifB6vF:b-896fix) ([raw file](https://github.com/pyomeca/bioptim/blob/162b77335c954b18cb9a8322861dec77b18219f5/bioptim/interfaces/biorbd_model.py#L15)):*
<details><summary><i>Previously, pariterre (Pariterre) wrote…</i></summary><blockquote>

'str' 'biorbd.Model'
</blockquote></details>

Done.
___
*[`tests/test_biorbd_model.py` line 25 at r1](https://reviewable.io/reviews/pyomeca/bioptim/605#-NMigazvlHFKgkTAW1Yt-r1-25:-NMjQolD1n2K_qlTGGVt:b-896fix) ([raw file](https://github.com/pyomeca/bioptim/blob/162b77335c954b18cb9a8322861dec77b18219f5/tests/test_biorbd_model.py#L25)):*
<details><summary><i>Previously, pariterre (Pariterre) wrote…</i></summary><blockquote>

`BiorbdModel([])` ?
</blockquote></details>

Done.

**Review by pariterre — APPROVED** [review]
LGTM

**Pull request merged**